### PR TITLE
MAKAIRA-3360 [Component] Dashboard App Menu

### PR DESCRIPTION
**Pull request by nfqphihungnguyen**

### CODE REVIEW

File: src/components/AppMenu/AppMenu.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import { Text } from '../Text/Text'
+
+interface AppMenuProps {
+  icon?: IconType
+  text?: string
+  badge?: React.ReactElement
+  onClick?: () => void
+}
+
+const AppMenu: React.FC<AppMenuProps> = ({ icon, text, badge }) => {
```
Comment (JanMaeg) [inline src/components/AppMenu/AppMenu.tsx:13]: Change the name of the component to `AppTitle`

File: src/components/AppMenu/AppMenu.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import styles from '@/components/AppMenu/AppMenu.module.scss'
+import { IconType } from 'react-icons'
+import { Text } from '../Text/Text'
+
+interface AppMenuProps {
+  icon?: IconType
+  text?: string
+  badge?: React.ReactElement
```
Comment (JanMaeg) [inline src/components/AppMenu/AppMenu.tsx:9]: Badge should only allow simple strings. We don't support and need passing components to the badge.

File: src/components/AppMenu/AppMenu.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+  return (
+    <div className={styles.wrapper}>
+      {getIcon()}
+      {text && (
+        <Text element="span" size="bravo" weight="bold" className={styles.text}>
+          {text}
+        </Text>
+      )}
+      {badge && <span className={styles.badge}>{badge}</span>}
```
Comment (JanMaeg) [inline src/components/AppMenu/AppMenu.tsx:34]: Styling of the badge is missing.

**Review by JanMaeg — CHANGES_REQUESTED** [review]

**Comment by MrAvantiC** [comment]
Closed in favour of https://github.com/MakairaIO/app-boilerplate-nextjs/pull/8

**Pull request closed without merging**